docs/markdown/star_formation.md

```diff
@@ -0,0 +1,33 @@
+# Star Formation Recipe 
+
+## Checking for Jeans Violation
+The star formation recipe is implemented via the Stochastic Stellar Population specialisation. Every cell is checked for Jeans violation and star particles are added if $\lambda_J = c_s/\sqrt{G\rho} < J \times dx$, where J is $0.5$. 
+
+Adding star particle to every Jeans violating cell can lead to a large number of small mass stars which can be cumbersome. To alleviate this issue, we control the star formation rate through $\epsilon_{\rm ff}$ and $\epsilon_{*}$. $\epsilon_{\rm ff}$ is the efficiency per freefall time given by $=\dot{M}_* \cdot t_{\rm ff} / M_{\rm cell} \cdot dt $, while $\epsilon_{*}$ is the fraction of cell mass that gets converted into stars. 
```

Comment:
It looks like there is a LaTeX formatting issue in this paragraph:
<img width="750" alt="Screenshot 2025-04-16 at 8 22 59 PM" src="https://github.com/user-attachments/assets/51c47cb2-cfe0-49bc-96ed-09a7719cc09d" />